content/engine/release-notes/25.0.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Docker Engine 25.0 release notes
+description: Learn about the new features, bug fixes, and breaking changes for Docker
+  Engine
+keywords: docker, docker engine, ce, whats new, release notes
+toc_min: 1
+toc_max: 2
+skip_read_time: true
+aliases:
+- /engine/release-notes/
+- /engine/release-notes/latest/
+- /release-notes/docker-ce/
+- /release-notes/docker-engine/
+---
+
+This page describes the latest changes, additions, known issues, and fixes for Docker Engine version 25.0.
+
+For more information about:
+
+- Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
+- Changes to the Engine API, see [Engine API version history](../api/version-history.md).
+
+## 25.0.0
+
+{{< release-date date="2023-09-28" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 25.0.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A25.0.0)
+- [moby/moby, 25.0.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A25.0.0)
+
+### New
+
+- Add OpenTelemetry tracing. [moby/moby#45652](https://github.com/moby/moby/pull/45652), [moby/moby#45579](https://github.com/moby/moby/pull/45579)
+- Add support for CDI devices under Linux. [moby/moby#45134](https://github.com/moby/moby/pull/45134), [docker/cli#4510](https://github.com/docker/cli/pull/4510), [moby/moby#46004](https://github.com/moby/moby/pull/46004)
+- Add a `--health-start-interval` flag to `docker container` and `docker service` commands. [docker/cli#4405](https://github.com/docker/cli/pull/4405)
+- Add a `--log-format` flag to `dockerd` to control the logging format: text (default) or JSON. [moby/moby#45737](https://github.com/moby/moby/pull/45737)
+- Add support for recursive read-only mounts. [moby/moby#45278](https://github.com/moby/moby/pull/45278)
+
+### Bug fixes and enhancements
+
+- API: Fix error message for invalid policies at `ValidateRestartPolicy` [moby/moby#46352](https://github.com/moby/moby/pull/46352)
+- API: Update `/info` endpoint to use singleflight. [moby/moby#45847](https://github.com/moby/moby/pull/45847)
+- Add additional interval for healthchecks during the start period. [moby/moby#40894](https://github.com/moby/moby/pull/40894)
```

Comment:
have a single entry for health check intervals?

Reply:
Yup, squash all related entries down IMO.